src/af.ui.js

```diff
@@ -853,28 +853,47 @@
             var forceRefresh=anchor.getAttribute("data-refresh");
 
             if(found.length>0){
-
                 if(forceRefresh){
                     that.showLoading&&that.showMask("Loading Content");
                     $.ajax(target).then(function(res){
-                        found.html(res);
+						if(!$(res).hasClass('panel')) {
+							// cmwong 20150404
+							// do not hv panel, just replace everything
+							// use append so that the js in panel will execute
+							found.empty().append(res);
+						} else {
+							// cmwong 20150404
+							// only the first panel
+							// only take childnodes under panel (do not want the panel node)
+							var $res=$.create("div",{html:res});
+							found.empty().append( $( $res.find(".panel")[0] ).html() );
+						}
                         that.showLoading&&that.hideMask();
-                        return that.loadContent("#"+found.prop("id"),newTab,back,transition,anchor);
+                        that.loadContent("#"+found.prop("id"),newTab,back,transition,anchor);
                     });
-                }
-                else
-                    return that.loadContent("#"+found.prop("id"),newTab,back,transition,anchor);
+					// cmwong 20150404
+					// return move here, if not it with fire 2 times ajax
+					return;
+                } else {
+                    that.loadContent("#"+found.prop("id"),newTab,back,transition,anchor);
+					return;
+				}
             }
             that.showLoading&&that.showMask("Loading Content");
             $.ajax(target).then(function(res){
                 var $res=$.create("div",{html:res});
-                if(!$res.hasClass(".panel")){
```

Comment:
1. '.panel' is wrong
2. $res.hasClass() only check on the first node, $res was already wrap with an div, it won't found any panel